spread.yaml

```diff
@@ -57,6 +57,24 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    external-ubuntu-core:
+        type: adhoc
+        environment:
+            SPREAD_ADHOC_UC_ADDRESS: "$(HOST: echo ${SPREAD_ADHOC_UC_ADDRESS:-localhost:8022})"
+        allocate: |
+            ADDRESS $SPREAD_ADHOC_UC_ADDRESS
+        systems:
+            # FIXME: remove the ubuntu-core-16.04-64 dummy system once the runner handles tasks without systems assigned
+            - ubuntu-core-16.04-64:
+                username: test
+                password: ubuntu
+            - ubuntu-core-16-device-amd64:
```

Comment:
Per the thread in this PR, I still don't quite see why this is necessary. It would be nice to reuse the ubuntu-core-16-64 system for that instead. Let's please catch up online to discuss it.